mmdet/core/evaluation/coco_utils.py

```diff
@@ -50,6 +50,14 @@ def coco_eval(result_files,
         cocoEval.accumulate()
         cocoEval.summarize()
 
+        predictions = cocoEval.cocoDt.imgToAnns
```

Comment:
probably it is worth using `result_types` to enable text detection metrics computation.
In common case we don't need to perform text detection quality estimation